### PR TITLE
Paloma: new chain

### PR DESCRIPTION
**Pull request by kinrokinro**

### CODE REVIEW

**Review by editaahn — COMMENTED** [review]
Hi! Could you remove all the coinGeckoId lines from your changes? Since the coinGeckoId doesn't exist yet, it's not acceptable for the registry.

**Comment by kinrokinro** [comment]
> Hi! Could you remove all the coinGeckoId lines from your changes? Since the coinGeckoId doesn't exist yet, it's not acceptable for the registry.

Done! @editaahn

File: cosmos/tumbler.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     {
       "coinDenom": "GRAIN",
       "coinMinimalDenom": "ugrain",
-      "coinDecimals": 6
+      "coinDecimals": 6,
```
Comment (editaahn) [inline cosmos/tumbler.json:27]: The comma has to be removed here.
```suggestion
      "coinDecimals": 6
```

File: cosmos/tumbler.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
   "stakeCurrency": {
     "coinDenom": "GRAIN",
     "coinMinimalDenom": "ugrain",
-    "coinDecimals": 6
+    "coinDecimals": 6,
```
Comment (editaahn) [inline cosmos/tumbler.json:45]: Here too!

**Comment by kinrokinro** [comment]
@editaahn should be fixed now, sorry for the hustle!

**Pull request merged**